openupgrade_scripts/scripts/sale_stock/17.0.1.0/post-migration.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2025 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Fill delivery_date and incoterm_location according
+    # https://github.com/odoo/odoo/blob/00818b7bfaf22635b0c40b3b9c7e37e0c9789da1/
+    # addons/sale_stock/models/account_move.py#L116-L134
+    openupgrade.logged_query(
```

Comment:
I think we should apply some WHERE conditions... I mean, what happens if you have a LOT of sale order lines performance-wise?

Reply:
This SQL is already optimized, so it won't suffer in case of a big number of lines. You can try in a DB with lots of them (or use the populate tool for it).